app/assets/stylesheets/views/_organisations.scss

```diff
@@ -146,3 +146,12 @@
 .organisation__address {
   font-style: normal;
 }
+
+.organisation__border-top {
```

Comment:
Not sure if this'll help or not, but the heading component can have a border-top of 1px if needed. Colour might be an issue though.

Reply:
Yeah, I noticed that, but the colour can only be passed via a brand, so it wouldn't work. Perhaps we can extend that component in the future, but think it works for now.